app/src/main/java/edu/nextstep/camp/calculator/MainActivity.kt

```diff
@@ -1,46 +1,41 @@
 package edu.nextstep.camp.calculator
 
 import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
+import com.soolee.domain.Calculator
 import edu.nextstep.camp.calculator.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
     private lateinit var binding: ActivityMainBinding
+    private lateinit var calculator: Calculator
 
+    lateinit var textView: TextView
```

Comment:
```suggestion
    lateinit var textView: TextView
```
멤버 변수가 많아질수록 클래스는 멤버 변수에 강하게 의존하게 됩니다.
이 멤버 변수를 사용하지 않고 구현해보면 어떨까요?

Reply:
이미 binding 이라는 변수가 있으니 이걸 사용해도 될 것 같네요!